jcl/src/java.base/share/classes/java/lang/Thread.java

```diff
@@ -700,7 +712,12 @@ public void interrupt() {
 		sun.nio.ch.Interruptible localBlockOn = blockOn;
 		if (localBlockOn != null) {
 			localBlockOn.interrupt(this);
-		}
+        }
+        /*[IF Java14]*/
+        if (!isAlive()) {
+			deadInterrupt = true;
```

Comment:
Formatting seems odd here too
```suggestion
		deadInterrupt = true;
```